snmp_exporter/collector.py

```diff
@@ -56,11 +56,16 @@ def collect_snmp(config, host, port=161):
   """Scrape a host and return prometheus text format for it"""
 
   start = time.time()
+  
+  community = 'public'
+  if (config.has_key('snmpcommunity')):
```

Comment:
config.get('snmpcommunity', 'public')

I think just 'community' would be sufficient, as it's implicit that it's SNMP.